README.md

```diff
@@ -300,9 +300,156 @@ additional data, using the MySQL Management Pack as an example.</summary>
+
+- [mp-test Troubleshooting Guide](doc/mp-test.md#troubleshooting)
+- [mp-init Troubleshooting Guide](doc/mp-init.md#troubleshooting)
+- [mp-build Troubleshooting Guide](doc/mp-build.md#troubleshooting)
+
+<details>
+<summary>
+<h2>Docker Issues</h2>
+</summary>
+
+<details>
+<summary>
+<h3> Cannot connect to docker daemon (Windows)?</h3>
+</summary>
+If you're having trouble getting Docker to run on your system, you can refer to the Docker documentation for instructions on how to start Docker on [macOS](https://docs.docker.com/docker-for-mac/install/), [Linux](https://docs.docker.com/desktop/install/debian/#launch-docker-desktop), and [Windows 10 and 11](https://docs.docker.com/desktop/install/windows-install/#start-docker-desktop).
+</details>
+
+<details>
+<summary>
+<h3> Permission denied while trying to connect to the Docker daemon?</h3>
+</summary>
+If you're having trouble with permissions on a Windows system, you can refer to the Docker documentation for instructions on how to [Understand permission requirements for Windows](https://docs.docker.com/desktop/windows/permission-requirements/).
+</details>
+
+</details>
+
+<details>
+<summary>
+<h2>Registry Issues:
+TODO:
+</details>
+
+<details>
+<summary>
+<h2>Dockerfile Issues:</h2>
+TODO:
+</details>
+
+<details>
+<summary>
+<h2>Adapter</h2>
+</summary>
+
+<details>
+<summary>
+<h3> Where are the adapter logs stored locally? </h3>
+</summary>
+Logs are generated and stored in the `logs` directory whenever the adapter runs locally using' mp-test'
+</details>
+
+<details>
+<summary>
+<h3> Where the adapter logs stored VMware Aria Operations?</h3>
+</summary>
+Logs are generated and stored in the cloud proxy at `$ALIVE_BASE/user/log/adapter/<ADAPTERNAME>_adapter3/<ADAPTER_INTERNAL_INSTANCE_ID>`. ADAPTERNAME should match the name of the adapter used in the manifest.txt, and the ADAPTER_INTERNAL_INSTANCE_ID should match the Internal ID found in VMware Aria Operations at Environment>Inventory>Adapter Instances>My Adapter Adapter Instance>Instance** in the rightmost column.
+</details>
+
+<details>
+<summary>
+<h3> What are the different log files?</h3>
+</summary>
+There are five types of log files: adapter, server, build, test, and validation logs. Each log file is prepended with the type of
+log file followed by a number that represents rollover.
+
+- server.log:
+Contains all logs related to the HTTP sever inside the container. Server logs can't be modified since the server code comes packaged
+inside the [base-adapter image](https://projects.registry.vmware.com/harbor/projects/46752/repositories/base-adapter/artifacts-tab) Python image.
+
+- adapter.log
+Contains all logs related to the adapter
+
+- test.log
+Contains all logs related to `mp-test`
+
+- build.log
+Contains all logs related to `mp-build`
+
+- validation.log
+Contains all logs related to the validation of the collection result(s).
```

Comment:
May be somewhat obvious, but it could be useful to mention that these are only found locally